lib/diagnostic.rb

```diff
@@ -105,15 +120,21 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < cat
+  include Carnivorous
+  def roar
+    p "ROAR!"
+  end
+end
 ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# Composition allows you to include functions that are simliar in objects that are
+# not neccessarily related.
 ##
```

Comment:
True! It also allows us to inherit behavior from multiple places instead of just from one as we do with inheritance!